### ISSUE TITLE
Unclear error message thrown on non-valid orderBy (e.g. non-array)

### ISSUE THREAD

**Issue by wesselvdv** [graphile/crystal#1854]
When this occurs you only get a single message: "Invalid!", quite difficult to track down where the issue is.

https://github.com/graphile/crystal/blob/818b0e619949a3e7a89fea16430f6b4e4bd726aa/graphile-build/graphile-build-pg/src/plugins/PgConnectionArgOrderByPlugin.ts#L238

Also, side note. Shouldn't `null` be a valid input? Or should it just be completely absent? It isn't a required input ofc.

**Comment by benjie** [comment]
Yes, `null` should be a valid orderBy of dubious usefulness. Why would you pass `null`?! Anyway yes this needs fixing; it's on the list.
